### DISCUSSION TITLE
Invalid copilot token: missing token: 403

### DISCUSSION THREAD

**Discussion by IKatsuba** [community/community#51675] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I am in Cyprus and working with a VPN through Germany. Copilot worked fine until today. I was trying to logout and login, there is no changes. Help me, please!
```
[INFO] [auth] [2023-04-01T13:06:16.653Z] Invalid copilot token: missing token: 403 
[ERROR] [default] [2023-04-01T13:06:16.655Z] GitHub Copilot could not connect to server. Extension activation failed: "At this time, Copilot is not available in your location"
```

**Comment by unspectd** [comment]
Many people including me have the same problem

**Comment by Akash1134** [comment]
Hi @IKatsuba ,

Thanks for getting in touch.

Due to current trade controls, your GitHub account has been restricted from making use of Copilot. This is a region-based restriction and applies to all users located in some trade-restricted countries, being flagged for the same earlier, and is separate from other GitHub services. You can read more about it here:

[GitHub and Trade Controls](https://docs.github.com/en/site-policy/other-site-policies/github-and-trade-controls)

Sorry if that wasn't what you were hoping to hear. 

If you feel this should not be the case, feel free to contact our Support team and [open a ticket ](https://support.github.com/request?tags=dotcom-footer)with them. Thanks!
